config.json

```diff
@@ -11,7 +11,7 @@
         "codecov"
     ],
     "webpackVersion": "latest",
-    "minNode": "4.8",
-    "latestNodeLTS": "6",
-    "latestNode": "8"
+    "minNode": "6.9.0",
```

Comment:
```
node: {
   "min": "6.9.0",
   "lts": "8.9.0",
   "stable/current": "9"
}
```

Reply:
I suppose we should just call these the same thing Node does.

```
node: {
   "maintLTS": "6.9.0",
   "activeLTS": "8.9.0",
   "current": "9"
}
```

Reply:
Changed on `next`